### DISCUSSION TITLE
Non Rigid ICP on Meshlab

### DISCUSSION THREAD

**Discussion by robertorocca** [cnr-isti-vclab/meshlab#892] [Ideas]
Hi,
i have seen the "Scalable non-rigid registration for multi-view stereo data" Project  on VCG website. Some hope to have "non rigid ICP" on Meshlab?
Thanks in advance.

**Comment by alemuntoni** [comment]
Sorry for the late reply! 
Right now I am working on big new feature that will allow to install new user-defined plugins into meshlab. With some good wiki-pages and documentation on how to implement a meshlab plugin (it is quite easy... what is missing is the documentation :) ), I hope that the community will contribute on developing new plugins (like this one) that can be loaded at runtime!
